src/components/search/Hit.tsx

```diff
@@ -74,7 +79,27 @@ export default function HitComponent({ hit }: { hit: SearchHit }): JSX.Element {
             </Flex>
           )}
         </Text>
+        <Button
+          rounded="md"
+          w="full"
+          mt={4}
+          py="7"
+          bg={useColorModeValue("blue.900", "blue.50")}
```

Comment:
```suggestion
          bg={useColorModeValue("brand.primary", "blue.50")}
```

we have been using the primary blue for CTAs so i think we should try to standardise on that 👍